src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.targets

```diff
@@ -453,6 +453,11 @@ Copyright (c) .NET Foundation. All rights reserved.
                                     Value="$(StartupHookSupport)"
                                     Trim="true" />
 
+    <RuntimeHostConfigurationOption Include="System.Resources.ResourceManager.AllowCustomResourceTypes"
```

Comment:
(nit) can you keep this list sorted? Put this by the other `System.Resources` setting above.